jobeet/.gitignore

```diff
@@ -12,3 +12,5 @@
 /src/Acme
 /src/AppBundle
 .idea/
+web/data
+*.php~
```

Comment:
ez tüneti kezelés...
ne írj gedit-ben és nem lesz ilyen, ez nem tartozik ide, illetve be lehet állítani a gedit-ben, hogy ne csinálja ezt